dev-docs/source/conf.py

```diff
@@ -11,16 +11,10 @@
 
 
 # -- General configuration ------------------------------------------------
-
-if LooseVersion(sphinx_version) > LooseVersion("1.6"):
-    def setup(app):
-        """Called automatically by Sphinx when starting the build process
-        """
-        # Add support for multiple Sphinx versions
-        if hasattr(app, "add_css_file"):
-            app.add_css_file("custom.css")
-        else:
-            app.add_stylesheet("custom.css")
+def setup(app):
+    """Called automatically by Sphinx when starting the build process
+    """
+    app.add_stylesheet("custom.css")
```

Comment:
This was renamed in 1.8 (although it wasn't removed until 4.0)
```suggestion
    app.add_css_file("custom.css")
```
https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_css_file

Reply:
Oh I was looking at the man page for 1.6 :man_facepalming: